README.md

```diff
@@ -13,92 +13,92 @@ Swarm Control Planes.
 
 - v11
   - v11.2
-    - [v11.2.1](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v11.2.1.md)
-    - [v11.2.0](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v11.2.0.md)
+    - [v11.2.1](https://github.com/giantswarm/releases/blob/master/aws/v11.2.1/release-notes.md)
```

Comment:
This is an issue - this will break all the links in `slack` we have pushed out to customers right now :(

Maybe symlink or something? Even though I doubt it works as we direct link to the `.md`.

Reply:
I was worried about that. Using links directly to GitHub ties us to this directory structure. I could keep the notes where they are. Or we could have a migration period. Or we could host the notes with a real website where we could do redirects though that wouldn't help this particular case.

Reply:
After some discussion, it sounds like broken links in Slack aren't a high priority.